monero-harness/src/lib.rs

```diff
@@ -27,7 +27,8 @@ pub use testcontainers::clients::Cli;
 use testcontainers::{Container, RunnableImage};
 use tokio::time;
 
-use monero::{Address, Amount};
+use monero_address::MoneroAddress;
```

Comment:
where is the serialization of this defined?

Reply:
Unchangingly in swap-serde/src/monero.rs, mod address